### PR TITLE
Fix cstdint with gcc 14.2.1.

### PR DESCRIPTION
**Pull request by gonsolo**

### CODE REVIEW

File: OpenEXR/config/LibraryDefine.cmake [hunk 1]
```diff
@@ -81,15 +81,15 @@ function(OPENEXR_DEFINE_LIBRARY libname)
...
-  install(TARGETS ${libname}
-    EXPORT ${PROJECT_NAME}
-    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
-    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-    INCLUDES DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
-    PUBLIC_HEADER
-      DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/${OPENEXR_OUTPUT_SUBDIR}
-  )
+  #  install(TARGETS ${libname}
```
Comment (mmp) [inline OpenEXR/config/LibraryDefine.cmake:84]: Are these changes intended? (And if so, what's the purpose?)

**Comment by mmp** [comment]
(I went ahead and merged the <cstdint> changes separately in ebd7bfa1a7e22b6cc0f2637b0aa9253f5a2f1dc8.)

File: OpenEXR/config/LibraryDefine.cmake (hunk 1 above)
Comment (gonsolo) [reply OpenEXR/config/LibraryDefine.cmake:84]: They are from your commit https://github.com/mmp/openexr/pull/1/commits/5cfb5dab6dfada731586b0281bdb15ee75e26782 but I have no idea how they ended up in the pull request.
Sorry.

**Comment by gonsolo** [comment]
> (I went ahead and merged the changes separately in ebd7bfa1a7e22b6cc0f2637b0aa9253f5a2f1dc8.)

Thanks.
